projects/canopy/src/lib/filter-buttons/filter-button/filter-button.component.html

```diff
@@ -0,0 +1,21 @@
+<input
+  (click)="onCheck()"
+  [attr.aria-describedby]="variant === 'select-multiple' ? ariaDescribedBy : null"
+  [attr.checked]="checked === true ? true : null"
+  [attr.disabled]="disabled === true ? '' : null"
+  [attr.id]="id"
+  [attr.name]="name"
+  [attr.value]="value"
+  class="lg-filter-button__input"
+  [type]="variant === 'select-multiple' ? 'checkbox' : 'radio'"
+/>
+
+<label [attr.for]="id" class="lg-filter-button__label">
+  <span class="filter-button__icon-wrapper">
+    <lg-icon class="filter-button__icon-add" name="add"></lg-icon>
+    <lg-icon class="filter-button__icon-subtract" name="subtract-spot-outline"></lg-icon>
+    <lg-icon class="filter-button__icon-selected" name="checkmark"></lg-icon>
```

Comment:
I think the `add` and `selected` icons could use the `checked` value to decided wether to be added to the DOM or not instead of always be present and handled by the css.

Reply:
Done.